src/main/java/com/emc/ecs/nfsclient/rpc/RpcWrapper.java

```diff
@@ -281,16 +265,14 @@ public String chooseIP(byte[] key) throws IOException {
      */
     private void callRpcChecked(S request, RpcResponseHandler<? extends T> responseHandler, String ipAddress)
             throws IOException {
-        if (_verbose) {
-            LOG.info("server: %s port: %s %s", new Object[] { _server, _port, request.toString() });
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("server {}, port {}, request {}", _server, Integer.toString(_port), request.toString());
```

Comment:
... this is where you could simply remove the isDebug call and have one line:
`LOG.debug("server {}, port {}, request {}", _server, _port, request);`
the conversions will happen automatically, but only if the template is rendered, so there's practically no overhead